t/t5601-clone.sh

```diff
@@ -68,6 +68,13 @@ test_expect_success 'clone respects GIT_WORK_TREE' '
 
 '
 
+test_expect_success MINGW,CASE_INSENSITIVE_FS 'core.worktree is not added due to path case' '
```

Comment:
Now that you no longer use `cygpath`, how about dropping the `MINGW` prereq?

Reply:
Done.